MediaBrowser.Model/Lyrics/TimeTag.cs

```diff
@@ -0,0 +1,28 @@
+namespace MediaBrowser.Model.Lyrics;
+
+/// <summary>
+/// TimeTag model.
+/// </summary>
+public class TimeTag
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="TimeTag"/> class.
+    /// </summary>
+    /// <param name="position">The character index of the lyric.</param>
+    /// <param name="timestamp">The timestamp the lyric is synced to.</param>
+    public TimeTag(int position, int timestamp = null)
+    {
+        Position = position;
+        Timestamp = timestamp;
+    }
+
+    /// <summary>
+    /// Gets the character index of the lyric.
+    /// </summary>
+    public int Position { get; }
+
+    /// <summary>
+    /// Gets the timestamp the lyric is synced to.
```

Comment:
What unit is the the timestamp in? (looks like it might be ticks?)

Reply:
Yeah, it's in ticks.

Reply:
I have changed the timestamp object, so it's now a _long_ type instead of an _int_, as this is what the other ticks do.

Reply:
I have also updated the description to say "in ticks'